microsoft/testsuites/performance/common.py

```diff
@@ -80,3 +80,25 @@ def cleanup_process(environment: Environment, process_name: str) -> None:
     for node in environment.nodes.list():
         kill = node.tools[Kill]
         kill.by_name(process_name)
+
+
+def format_disk(
+    node: Node,
+    disk_names: List[str],
+) -> List[str]:
+    fdisk = node.tools[Fdisk]
+    partition_disks: List[str] = []
+    for data_disk in disk_names:
+        fdisk.delete_partitions(data_disk)
+        partition_disks.append(fdisk.make_partition(data_disk, format=False))
+    return partition_disks
+
+
+def stop_raid(node: Node) -> None:
+    mdadm = node.tools[Mdadm]
+    mdadm.stop_raid()
+
+
+def make_raid(node: Node, disk_list: List[str]) -> None:
```

Comment:
It looks every time the `stop_raid` is called before `make_raid`. How about change this method to `recreate_raid` and call `stop_raid`?

Reply:
Makes sense! I will update this

Reply:
Though I feel adding "re" prefix before the method name might be a bit wordy. How about we let the method name be `make_raid` and I can add `stop_raid` at the beginning of this method?

Reply:
The "re" means if exists, then removed. Sometimes people may want something, and it's ok if exists.